prometheus-exporters/vrops-exporter/alerts/virtualmachine.alerts

```diff
@@ -32,5 +30,3 @@ groups:
       meta: "Disk almost full with over 90% for {{ $labels.virtualmachine }}. Please increase disk size."
       playbook: docs/devops/alert/vcenter/
     annotations:
-      description: "Disk almost full with over 90% for {{ $labels.virtualmachine }}. Please increase disk size."
-      summary: "Disk almost full with over 90% for {{ $labels.virtualmachine }}. Please increase disk size."
```

Comment:
Why did you remove this?

Reply:
not sure it got left out. Added it again -Thanks